src/net/sourceforge/kolmafia/textui/command/AbsorbCommand.java

```diff
@@ -48,14 +48,14 @@ public void run(final String cmd, String parameters) {
 
     // If not in inventory, try to retrieve it (if it's in inventory, doesn't matter if outside
     // Standard)
-    if (!InventoryManager.hasItem(match, true)
-        && !InventoryManager.retrieveItem(match)
-        && match.getCount(KoLConstants.inventory) == 0) {
+    if (!InventoryManager.retrieveItem(match) && match.getCount(KoLConstants.inventory) == 0) {
```

Comment:
```suggestion
    if (match.getCount(KoLConstants.inventory) == 0 && !InventoryManager.retrieveItem(match)) {
```
if the item is in the inventory already, no need to attempt to retrieve it. I suspect this is what the original author thought "`hasItem`" did.

Reply:
Ah wait, shouldn't that be `match.getCount(KoLConstants.inventory) < count` instead?

Reply:
Seems so, commited the change.

Reply:
good spot